user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -761,6 +762,21 @@ def __build_global_report_summary(self,
         output_files_info = JSONPropertiesContainer(output_files_raw, file_load=False)
         unsupported_ops_obj = UnsupportedOpsStageDuration(self.ctxt.get_value('local', 'output',
                                                                               'unsupportedOperators'))
+        rapids_output_dir = self.ctxt.get_rapids_output_folder()
+        stages_info_file = FSUtil.build_path(rapids_output_dir,
+                                             self.ctxt.get_value('toolOutput', 'csv',
+                                                                 'stagesInformation', 'fileName'))
+        stages_df = pd.read_csv(stages_info_file)
+
+        # Generate the statistics report
+        stats_report = SparkQualificationStats(
+            unsupported_operators_df=unsupported_ops_df,
+            stages_df=stages_df,
+            output_file=output_files_info.get_value('statistics', 'path'),
```

Comment:
nit: Can we pass the entire statistics output conf? We can refer the keys `path` and `columns` in the `SparkQualificationStats` class?

Reply:
Thanks @parthosa ! I have updated it to pass `ctxt` and referring all the filenames through the ctxt. PTAL.